src/BalanceClient.ts

```diff
@@ -0,0 +1,22 @@
+const {providers} = require("near-api-js");
+
+export class BalanceClient {
+
+  static async balanceOfOnNear(contractId: string, accountId: string): Promise<number> {
+    const connectionInfo = { url: "https://rpc.testnet.near.org" };
```

Comment:
could you extract this connection info to a const (or connectionInfo file) as it will be easier to add mainnet connection info (and other envs if needed)
You could then have this info in the constructor of the BalanceClient or some other way passed here

Reply:
Good point, this is also extracted to Near config file